spec/models/pawn_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Pawn, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'can move vertically and two squares on first move' do
+    game = Game.create
+    pawn = Pawn.create(game_id: game.id, x_position: 3, y_position: 1, color: 'white')
+
+    expect(pawn.first_move?(2)).to be true
+    expect(pawn.first_move?(3)).to be true
+  end
+
+  it 'can not move two squares after first move' do
+    game = Game.create
+    pawn = Pawn.create(game_id: game.id, x_position: 3, y_position: 3, color: 'white')
+
+    expect(pawn.standard_move?(3, 5)).to be false
+  end
+
+  it 'can not move horizontally' do
+    game = Game.create
+    pawn = Pawn.create(game_id: game.id, x_position: 2, y_position: 3, color: 'white')
+
+    expect(pawn.valid_move?(3, 3)).to be false
+    expect(pawn.valid_move?(1, 3)).to be false
+  end
+
+  it 'can only move diagonally if capturing' do
+    game = Game.create
+    pawn1 = Pawn.create(game_id: game.id, x_position: 2, y_position: 5, color: 'white')
+    pawn2 = Pawn.create(game_id: game.id, x_position: 3, y_position: 6, color: 'black')
```

Comment:
Rubocop is complaining about this line here because you're creating a new pawn instance and assigning it to the variable `pawn2` (`pawn2 = Pawn.create(...)`) however you don't ever do anything with `pawn2` anywhere else, so it's a "useless assignment" (Rubocop's term for this kind of thing).  The solution's actually pretty simple, just don't save it to a variable:

```ruby
pawn1 = Pawn.create(game_id: game.id, x_position: 2, y_position: 5, color: 'white')
Pawn.create(game_id: game.id, x_position: 3, y_position: 6, color: 'black')
```